common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -177,31 +252,98 @@ def login_and_verify_visible_problems(username, email, expected_problems):
         )
 
         login_and_verify_visible_problems(
+            self.cohort_verified_student_username, self.cohort_verified_student_email,
+            [self.group_verified_problem, self.visible_to_all_problem],
+            'verified'
+        )
+
+        login_and_verify_visible_problems(
+            self.cohort_audit_student_username, self.cohort_audit_student_email,
+            [self.group_audit_problem, self.visible_to_all_problem],
```

Comment:
Odd, I would have expected the default mode to be Audit. But if it is Honor, can you please change the course to have Honor and Verified as the 2 course modes (or enroll these other students in Audit)? For someone just reading the tests, it's very confusing to see that some students see neither the Audit nor the Verified content (since this is not a case that happens in "real life").